main.cpp

```diff
@@ -59,85 +59,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       ball
+//  action 1:   ball bounces
+ball.bounce();
+//  action 2:   ball crashes
+ball.crashes();
+//  action 3:   ball stops rolling
+ ball.stopRolling();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       plant
+//  action 1:   plant oxygenates
+plant.oxygenates();
+//  action 2:   plant produces CO2
+plant.produceCO2();
+//  action 3:   plant grows
+ plant.grow();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       cellphone
+//  action 1:   cellphone vibrates
+cellphone.vibrate();
+//  action 2:   cellphone rings
+cellphone.ring();
+//  action 3:   celphone warms up
+cellphone.warmUp();
```

Comment:
`cellphone` should be spelled with camelCase
- [ ] use camelCase with variable names.